src/Component/Symfony/Bundle/Routing/Configuration.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Symfony\Bundle\Routing;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('routing');
+        /** @var ArrayNodeDefinition $rootNode */
+        $rootNode = $treeBuilder->getRootNode();
+
+        $rootNode
+            ->children()
+                ->scalarNode('alias')->isRequired()->cannotBeEmpty()->end()
+                ->scalarNode('path')->defaultValue(null)->end()
+                ->scalarNode('identifier')->defaultValue('id')->end()
+                ->arrayNode('criteria')
+                    ->useAttributeAsKey('identifier')
+                    ->scalarPrototype()
+                    ->end()
+                ->end()
+                ->booleanNode('filterable')->end()
+                ->variableNode('form')->cannotBeEmpty()->end()
+                ->scalarNode('serialization_version')->cannotBeEmpty()->end()
+                ->scalarNode('section')->cannotBeEmpty()->end()
+                ->scalarNode('redirect')->cannotBeEmpty()->end()
+                ->scalarNode('templates')->cannotBeEmpty()->end()
+                ->scalarNode('grid')->cannotBeEmpty()->end()
+                ->booleanNode('permission')->defaultValue(false)->end()
+                ->arrayNode('except')
+                    ->scalarPrototype()->end()
```

Comment:
Is it within PR scope to switch from scalar to enum for both `except` and `only`?

https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/src/Sluggable/Util/Urlizer.php

Reply:
This is interesting comments, I think we can add some issue in a board with that kind of comment.